### DISCUSSION TITLE
Брать настройки для Auto из самого бризера (облака)

### DISCUSSION THREAD

**Discussion by dima81rus** [RealLord/tion_home_assistant#3] [Ideas]
_Предлагаю перенести дискуссию сюда, из основного проекта, чтобы не вносить там сумятици_

Есть вопрос\идея: а мы можем просто включать бризер в автоматический режим, а настройки брать из того что уже заложено в бризер? в текущей версии (1.06) если я передаю просто авто без каких то параметров в бризер - он принудительно выставляет 600:1-6, когда как в приложении у меня было уже преднастреоно 800:0-2.

Раньше, если я передавал просто auto, интеграция похоже не пыталась выставить не каких параметров и бризер включался на то что преднастроено в облаке.

**Comment by dima81rus** [comment]
попробую всё же вернуться к этому вопросу. возможно таки реально, как то сделать, чтобы мы просто передавали в бизер параметр auto, а сами настройки брались из бризера? раньше, в старой интеграции, так и было. теперь же принудительно передаётся 600:1-6, даже если я уже включил бризер в авто, настроил его в приложении и снова передал авто - всё одно ставится 600:1-6.

т.е. мысль то простая, попробовать бризеру передать auto без параметров и возможно он возьмёт настройки из облака.

**Reply by Rus-Senator** [reply]
Здравствуйте!

Я предлагаю просто сделать возможность из задавать где-то в настройках интеграции или даже выделить файл настроек отдельный. У каждого свои хотелки в плане Co2 и других параметров.
